cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -203,6 +209,12 @@ data ClientCommand
     Address
     (NonEmpty TxOut)
     CommonCLI
+  | SpendUTxO
+    NewTxFile
```

Comment:
These should be annotated similarly to `IssueDelegationCertificate` above.

Reply:
@Jimbo4350, done.